examples/basic/package.json

```diff
@@ -1,41 +1,34 @@
 {
   "name": "VideoPlayer",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "private": true,
   "scripts": {
     "android": "react-native run-android",
     "ios": "react-native run-ios",
-    "windows": "react-native run-windows",
+    "lint": "eslint .",
     "start": "react-native start",
-    "test": "jest",
-    "lint": "eslint ."
+    "test": "jest"
   },
   "dependencies": {
-    "@react-native-picker/picker": "^1.9.11",
+    "@react-native-picker/picker": "react-native-picker/picker",
```

Comment:
strange, what does it do ?

Reply:
I really don't know 😅 i will remove it ...